File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        s = s.toUpperCase();
        boolean answer = true;
        int pcnt = 0;
        int ycnt = 0;
        
        for(int i = 0;i<s.length();i++){
            if(s.charAt(i)=='Y') ycnt++;
            else if(s.charAt(i)=='P') pcnt++;
        }
        if(pcnt==ycnt) answer = true;
        else answer = false;
        
        System.out.println("Hello Java");
        
        
        return answer;
    }
}

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    int n = 0;
    for(int i=0; i<s.size(); i++){
        if(s[i] == 'p' || s[i] == 'P') n++;
        else if(s[i] == 'y' || s[i] == 'Y') n--;
    }
    return n==0;
}